hathor/event/event_manager.py

```diff
@@ -42,9 +42,63 @@
     HathorEvents.REORG_FINISHED,
     HathorEvents.TX_METADATA_CHANGED,
     HathorEvents.BLOCK_METADATA_CHANGED,
+    HathorEvents.CONSENSUS_TX_UPDATE,
+    HathorEvents.CONSENSUS_TX_REMOVED,
 ]
 
 
+def _todo(data: Dict[str, Any]) -> Dict[str, Any]:
+    raise NotImplementedError('TODO')
+
+
+def _empty(data: Dict[str, Any]) -> Dict[str, Any]:
+    return {}
+
+
+def _extract_tx(data: Dict[str, Any]) -> Dict[str, Any]:
+    tx = data['tx']
+    return {
+        'hash': tx.hash_hex,
+        # TODO: the rest
```

Comment:
Are you leaving this TODO intentionally?

Reply:
I'll expand the message, but I think this is out of the scope of this PR and it's perfectly fine to be left as a TODO for a later PR in the project.